superdesk/io/commands/remove_expired_content.py

```diff
@@ -36,25 +33,31 @@ def run(self, provider_name=None):
                 self.remove_expired(provider)
 
     def remove_expired(self, provider):
+        lock_name = 'ingest:gc'
+
+        if not lock(lock_name, expire=300):
```

Comment:
To avoid running multiple `ingest:gc` task at the same time it would be better to use `lock(lock_name, '', expire=300)` with no host parameters.